test/presubmit-tests.sh

```diff
@@ -50,10 +50,8 @@ function post_build_tests() {
   fi
 }
 
-# Run the unit tests with an additional flag '-mod=vendor' to avoid
-# downloading the deps in unit tests CI job
 function unit_tests() {
-  report_go_test -race -mod=vendor ./... || failed=1
+  report_go_test -race ./... || failed=1
```

Comment:
Can we keep `-mod=vendor` to the commands and add it where missing ? 

It's better to be explicit about the mode we want to run then to rely on the default of the go runtime. If you only see this line you have no idea whether vendoring is used or not, but you have to check which go version is used. 

Adding `-mod=vendor`  or `-mod=mod` everywhere is much better than relying on defaults.